hack/test-cmd.sh

```diff
@@ -133,16 +133,16 @@ openshift start \
   --images="${USE_IMAGES}"
 
 # validate config that was generated
-[ "$(openshift ex validate master-config ${MASTER_CONFIG_DIR}/master-config.yaml 2>&1 | grep SUCCESS)" ]
-[ "$(openshift ex validate node-config ${NODE_CONFIG_DIR}/node-config.yaml 2>&1 | grep SUCCESS)" ]
+[ "$(oadm validate master-config ${MASTER_CONFIG_DIR}/master-config.yaml 2>&1 | grep SUCCESS)" ]
```

Comment:
Do we still want to expose these as first-level commands, or move them to `openshift diagnostics -d NodeConfig`, etc as @liggitt mentioned?